### ISSUE TITLE
Double plugin start error

### ISSUE THREAD

**Issue by Sikatsu** [VolmitSoftware/React#240]
Full log: https://pastebin.com/A3GgNq3D

```
15.03 11:28:26 [Server] WARN Task #179 for React v6.2 generated an exception
15.03 11:28:26 [Server] INFO java.util.ConcurrentModificationException: null
15.03 11:28:26 [Server] INFO at java.util.ArrayList$Itr.checkForComodification(ArrayList.java:901) ~[?:1.8.0_131]
15.03 11:28:26 [Server] INFO at java.util.ArrayList$Itr.next(ArrayList.java:851) ~[?:1.8.0_131]
15.03 11:28:26 [Server] INFO at com.volmit.react.ReactPlugin$4.run(ReactPlugin.java:139) ~[React-6.2.jar:?]
15.03 11:28:26 [Server] INFO at com.volmit.react.util.Task$2.run(Task.java:55) ~[React-6.2.jar:?]
15.03 11:28:26 [Server] INFO at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftTask.run(CraftTask.java:71) ~[my.jar:git-Spigot-c3093ef-7e94e65]
15.03 11:28:26 [Server] INFO at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:353) [my.jar:git-Spigot-c3093ef-7e94e65]
15.03 11:28:26 [Server] INFO at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:739) [my.jar:git-Spigot-c3093ef-7e94e65]
15.03 11:28:26 [Server] INFO at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:406) [my.jar:git-Spigot-c3093ef-7e94e65]
15.03 11:28:26 [Server] INFO at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:679) [my.jar:git-Spigot-c3093ef-7e94e65]
15.03 11:28:26 [Server] INFO at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:577) [my.jar:git-Spigot-c3093ef-7e94e65]
15.03 11:28:26 [Server] INFO at java.lang.Thread.run(Thread.java:748) [?:1.8.0_131]
```

**Comment by skyybender** [comment]
Same problem here. It comes with the latest version of React (6.2) and will result in a plugin-crash. Better do not instal the latest update until there is a fix :)

**Comment by cyberpwnn** [comment]
There is no crash. A crash is when the server shuts down abruptly.

The plugin runs fine, its just printing the exception to the console.

**Comment by skyybender** [comment]
No server crash but there is no reaction on any /react commands so...

**Comment by cyberpwnn** [comment]
Implemented in 6.2.1
